service/history/fx.go

```diff
@@ -182,11 +182,13 @@ func TelemetryInterceptorProvider(
 	logger log.Logger,
 	namespaceRegistry namespace.Registry,
 	metricsHandler metrics.Handler,
+	configuration *configs.Config,
```

Comment:
nit: just to be consistent with other places
```suggestion
	serviceConfig *configs.Config,
```